pkg/machines/components/networks/createNetworkDialog.jsx

```diff
@@ -403,10 +403,16 @@ class CreateNetworkModal extends React.Component {
             const ipv6 = ip === "IPv4 only" ? undefined : this.state.ipv6;
             const ipv4 = ip === "IPv6 only" ? undefined : this.state.ipv4;
             const netmask = utils.netmaskConvert(this.state.netmask);
+            // If you take an IP address an apply the network mask (bitwise and) then you get the network address - no host can have such address
+            // Increment by 1 to get the first usable address of the block
+            const ipv4FirstUsableAddress = (ipv4 &&
+                ipv4.split(".")
+                        .map((item, index) => (item & netmask.split(".")[index]) + Number(index == 3))
+                        .join("."));
```

Comment:
Could this helper function maybe be moved into `pkg/machines/components/networks/utils.js`?

Reply:
I don't reuse it anywhere but I can move it, sure.